packages/shared/src/modules/ai-assistant/libs/types/balance-wheel-analysis-response.dto.type.ts

```diff
@@ -0,0 +1,18 @@
+import { type QuizScoresGetAllItemResponseDto } from "../../../quiz/quiz.js";
+
+type SimplifiedQuizScoreDto = Omit<
+	QuizScoresGetAllItemResponseDto,
+	"createdAt" | "id" | "score" | "updatedAt" | "userId"
+>;
```

Comment:
It's hard to understand what this type is. Maybe create one from scratch?

Reply:
I tried to use already created types.